src/components/Quantity/Quantity.jsx

```diff
@@ -1,11 +1,26 @@
 import { useState } from "react";
+import React from "react";
 import Input from "../Input/Input";
 import Button from "../Button/Button";
 
 import "./Quantity.scss";
 
+export const AuthContext = React.createContext({});
+
+export const AuthPorvider = (props) => {
+  const cart = {
+    quantity: 1,
+  };
+
+  return (
+    <AuthContext.Provider value={{ cart }}>
+      {props.children}
+    </AuthContext.Provider>
+  );
+};
+
 export default function Quantity() {
-  const [inputState, setInputState] = useState(null);
+  const [inputState, setInputState] = useState(0);
 

```

Comment:
in here you will consume your Context api,
1. import `CartContext`
2. use `React.useContext` then passed the `CartContext` imported as argument: `React.useContext(CartContext)`
3. this context will return to you what we have passed in CartContext.Provider value in `App.js` (quantity and setQuantity){
4. retrieve that function and value like so: `const {quantity, setQuantity} = React.useContext(CartContext);`
5. Now you have the value of the quantity and the way to update this value